src/app/(BottomTabNavigation)/Profile/styles.ts

```diff
@@ -105,9 +105,10 @@ export default StyleSheet.create({
   },
   signOutButton: {
     alignSelf: 'flex-start',
-    width: 125,
+    width: 320,
     height: 45,
-    marginLeft: 27,
+    marginLeft: 29,
+    marginRight: 27,
```

Comment:
I'm trying to get our team trending away from using margin values, as they don't scale well on different screens. I'd recommend using flex properties to align the items within the `TouchableOpacity`. It may be helpful to put the "Log out" text and icon within a sub-View, and move it around within the box using the "justify-content" and "align-items" CSS properties.

Reply:
I went through and got rid of left/right margin for everything except what's contained in the actions container and replaced with percentage width. Working on adjusting the log out button now. Any ideas that won't require using margin/padding?

Reply:
Also noticed the StyledButton Component --> Is there a possibility to created a modified component that matches the style for the logout button with the icon, text, and arrow? Or should I just work within the profile styling files

Reply:
I'd say just work within the profile styling files. That way you gain some experience styling. We can standardize components and everything later.

Reply:
In this case, I think "paddingHorizontal" will probably be fine, if it's around 20 px (you can check the spacing using figma dev mode). But within those bounds, try using the `justifyContent: "space-between"` option, which will push all elements within the `TouchableOpacity` as far away from each other as possible.

Reply:
We also want to include the "right-caret.svg" icon in the logout button. Observe how the `SignOut` icon is imported and used and try to replicate it.

<img width="836" alt="Screen Shot 2024-03-02 at 4 46 53 PM" src="https://github.com/calblueprint/impact-fund/assets/66931067/57279907-fc4c-4b74-9ceb-98882d3229e3">

Reply:
One last thing, if we want the the "Log out" text and icon to appear together and on the left side, how could we "group" them?